cmd/bucket-targets.go

```diff
@@ -381,9 +384,19 @@ func (sys *BucketTargetSys) getRemoteTargetClient(tcfg *madmin.BucketTarget) (*T
 		StorageClass:        tcfg.StorageClass,
 		disableProxy:        tcfg.DisableProxy,
 		healthCancelFn:      cancelFn,
+		Arn:                 tcfg.Arn,
+		ID:                  arnID(tcfg.Arn),
+		ResetID:             tcfg.ResetID,
 	}
 	return tc, nil
 }
+func arnID(arn string) string {
+	slc := strings.Split(arn, ":")
+	if len(slc) != 6 {
+		return ""
+	}
+	return slc[4]
+}
```

Comment:
all this will have been properly validated before it is used here - however, I am removing this function as the target ID is not really used anywhere.